awesome_owl/static/src/card/card.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+
+import { Component, useState } from "@odoo/owl";
+
+export class Card extends Component {
+    static template = "awesome_owl.card";
```

Comment:
So, yes `static template = "awesome_owl.playground";` should be `static template = "awesome_owl.Playground";` this mistake is in the tutorial code and not in yours.
```suggestion
    static template = "awesome_owl.Card";
```